src/alto/app/estimator.py

```diff
@@ -0,0 +1,76 @@
+"""

```

Comment:
Do you mind explaining what do the folder names 'app' and 'model' mean under the src/alto path? What modules and functions do we plan to include under these two folders? thks.